app/views/compute_resources_vms/form/azurerm/_volume.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= number_f f, :disk_size_gb,
+             {
+               :class    => "col-md-2",
+               :label    => _("Size (GB)"), :label_size => "col-md-2",
+               :required => true,
+               :min      => 0,
+               :value    => f.object.disk_size_gb || f.object.disk.disk_size_gb,
+               :help_inline => _("Additional number of disks can be added based on VM Size. For more details, please refer to Microsoft Azure's documentation")
+             }
+%>
+
+<%= selectable_f f, :data_disk_caching, options_for_select(["None", "ReadOnly", "ReadWrite"],
+                    :selected => f.object.data_disk_caching || f.object.disk.caching),
+                 { :include_blank => _('Please select Data Disk Caching method') },
```

Comment:
I think blank value means Azure default isn't it?

Reply:
Yeah but, this is just for a better user experience that we give the header for the list, makes sense?

Reply:
The UI is not explaining the user that the field can be left blank, and the default will be given.
IMHO `Default value will be determined by Azure` or something similar is better for the blank option.

Reply:
Ohkk, right. So, we might remove the `include_blank` option completely?